README.md

```diff
@@ -39,40 +36,34 @@ jobs:
 
 ### `enforce-sqg`
 
-If set to `true`, forces to complete execution successfully even if the failures conditions (SQG criteria) you have set are met.
+If set to `true`, the task with return a failure when security quality gates (SQG) criteria have failed.
+If set to `false`, the action reports SQG failures scenarios without enforcing them (i.e. give a grace period to development teams before you start breaking builds).
 
-This parameter can be useful if you want to detect SQG failures scenarios without enforcing them (i.e. give a grace period to development teams before you start breaking builds).
-
-Default is `false`.
-
-Implementation: get the SQG status from audit report - 
+Default is `false`.  
 
 ### `log-level`
 
-Sets the level of details in the logs, one of: `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`. 
+Sets the level of details in the action logs, one of: `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`. 
 Default is `INFO`.
 
 ### `data-enrich`
 
-Enrichs the OpenAPI file leveraging the default data dictionary. For each property with a standard format (such as uuid or date-time), patterns and constraints will be added to the OpenAPI file before it is audited. 
+Enrichs the OpenAPI file leveraging 42Crunch default data dictionary. For each property with a standard format (such as uuid or date-time), patterns and constraints will be added to the OpenAPI file before it is audited.
+
 Default is ` true`.
 
 ### `sarif-report`
 
-Converts the audit raw JSON format to SARIF and save the results into a specified file. 
+Converts the audit raw JSON format to SARIF and saves the results into a specified file.
 If not present, the SARIF report is not generated.
 
-> **Special case**: when an OpenAPI has no issues (clean report), we must still export a SARIF file, similar to [this one](../01.documentation/z.Assets/7-Security_Audit/cleanreport.sarif) so that Microsoft can distinguish between clean reports and no reports (i.e. the task has not run).
->
-> Important details is that report must contain this empty result array : `"results": []`
-
 ### `export-as-pdf`
 
-Exports the audit report as PDF, based on the already existing format documented [[here](https://github.com/42c-presales/automation-scripts/tree/main/audit-reports-as-html)] .
+Exports the audit report highlights as PDF.
```

Comment:
"Exports the highlights of the audit report as a PDF."

tbf, "highlights" sounds bit odd here, since it's a positive word and most of the time the results from Security Audit are negative. I might use "a summary" instead of "the highlights", it's more neutral so would work eve with crappy APIs :)